packages/terra-site/src/examples/status-view/StatusViewTemplate.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StatusView from 'terra-status-view';
+import _ from 'lodash';
```

Comment:
I'm a bit curious what impact this has on our bundle size. It looks like it is only being used in this file for unique ids. [ShortID](https://github.com/dylang/shortid) is another option that we could use that should result in a smaller bundle.